### ISSUE TITLE
"Reset" button is not working

### ISSUE THREAD

**Issue by peddiashrith** [sounishnath003/450-DSA-Tracker#2]
After marking some problems as done, I am unable to reset my progress after clicking the **reset** button

**Comment by sounishnath003** [comment]
RESET button to reset the Search Text. I name it wrong. It should be "CLEAR" instead of "RESET".

What do you think? 
RESET DB a button needed? @peddiashrith

**Comment by peddiashrith** [comment]
I think a **reset** button would be very helpful if he/she wants to continue from fresh.

**Comment by vaibhavmehra9** [comment]
@sounishnath003 Instead of "RESET" we can add a cross button in the search bar and also append the search query into the URL.

**Comment by ravireddy07** [comment]
Is anyone working on this issue....?

**Comment by sounishnath003** [comment]
@ravireddy07
till now no one!. I actually need to name the button `Clear` instead of `RESET`. If you want to work for any of thease. Welcome

**Comment by ravireddy07** [comment]
Yeah sure, I would like to take this issue..
Let me clear few things,
In below image, The reset button is to reset that filter(search filter) right..?
So, This issue is to add ne clear button to clear the selection of problems. am i right..? @sounishnath003 @peddiashrith 

![Screenshot (32)](https://user-images.githubusercontent.com/26524467/109605678-8c355080-7b4b-11eb-9061-38e6b4487028.png)

**Comment by sounishnath003** [comment]
@ravireddy07 
Yes!! You are correct, Actually, you can rename the Button to `CLEAR` and add a button that resets all the current progress of that particular QTopic like - `arrays`, `matrix` etc...

**Comment by ravireddy07** [comment]
Ok, I'll look Forward.

**Comment by sounishnath003** [comment]
@ravireddy07 

Is there any update regarding the RESET feature?

**Comment by ravireddy07** [comment]
Hey @sounishnath003 ,
Sorry regarding this time.. was busy with office work. 
Is that fixed now..?
